### ISSUE TITLE
Videos not playing

### ISSUE THREAD

**Issue by perfectsoundandvideo** [quelea-projection/Quelea#377]
Videos/media/audio is not playing. I can preview them and send them live but when I hit play nothing happens. The time slider doesn't move but I can move it manually (but that doesn't do anything either).

**Comment by berry120** [comment]
Hmm, works ok for me. What version are you using? Could also be worth reinstalling VLC, as that's what Quelea uses for multimedia playback.

**Comment by perfectsoundandvideo** [comment]
The latest nightly version that was released about 4 days ago. I installed VLC from their website and Quelea still gives me the error that VLC isn't installed. Also, side bug, Quelea never detects VLC when I install it from the Microsoft Store.

**Comment by berry120** [comment]
Hmm, do you get the same issue with the latest beta? https://github.com/quelea-projection/Quelea/releases/tag/v2021.0-beta-2 Seems to work ok for me on that release too, just want to check if it does your end.

**Comment by perfectsoundandvideo** [comment]
Yes, however interesting thing; it didn't always happen- it worked for a while on that build, and then it just randomly stopped.

**Comment by berry120** [comment]
That's odd, there's nothing on Quelea's side that should just break it suddenly. Is it possible you changed anything on the VLC side? You can also try removing or renaming the `.quelea` folder in `C:\users\USERNAME\.quelea` (or the alternative folder if on a non-windows system) and see if reverting back to a clean install helps.

**Comment by perfectsoundandvideo** [comment]
As I mentioned previously, VLC is installed, but Quelea can't find it. Is there a way to manually specify the path to VLC?

**Comment by berry120** [comment]
Ah sorry, missed that.

At present there's no way to specify it manually I'm afraid - the path in windows is given by a registry key (by default `HKEY_LOCAL_MACHINE\SOFTWARE\VideoLAN\VLC\InstallDir`.) Could you check to see if that points to the correct place?

**Comment by perfectsoundandvideo** [comment]
Thanks, sorry I didn't get back sooner, but I looked in my registry and it's not there. (VLC)

**Comment by berry120** [comment]
Hmm, in that case it must have been installed or updated via some means that doesn't have that key in place, so VLCJ (and therefore Quelea) can't find it.

At some point I'll have a look and see if there's anything I can work out with this, but afraid it's not something we'll have in place for 2021.0. Hopefully 2022.0 - and I'm aiming to have that release out in early 2022, all being well.

**Comment by berry120** [comment]
Sorry, one other thing I should have noted to start with - have you got VLC 64 bit installed? You may have the 32 bit version, if so then it won't work with Quelea since it's a 64 bit application.

**Comment by perfectsoundandvideo** [comment]
Thanks! That fixed it. I didn't realize I had downloaded the 32-bit version.

**Comment by berry120** [comment]
@perfectsoundandvideo Thanks for confirming!
